content/guides/identity-management/quick-start/3-configure-box.md

```diff
@@ -0,0 +1,89 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Configure Box
+
+Once we have created a login experience with Okta we need to have a Box
+application available that will permit us to use the Box APIs to search for and
+create users that are associated with the Okta user account.
+
+## Set up a Box app
+
+<Grid columns='2'>
+  <Choose option='box.app_type' value='create_new' color='blue'>
+    # Create a new Box app
+
+    Create and configure a new Box JWT application to start with a clean user list.
+  </Choose>
+
+  <Choose option='box.app_type' value='use_own' color='blue'>
+    # Use an existing approved app
+
+    Use one of your existing admin approved Box JWT applications from the Box developer
+    console.
+  </Choose>
+</Grid>
+
+<Choice option='box.app_type' value='create_new' color='none'>
+  # Create a new Box app
+
+  To create a new Box application that may be used to call the Box APIs, use
+  the following steps.
+
+  1. Go to the [Developer Console][devconsole]
+  1. Select **Create New App**
+  1. Select **Custom App** as the type of application to create, and click
+     **Next**
+  1. Select **OAuth 2.0 with JWT** as the authentication method, and click
+     **Next**
+  1. Give your Box app a unique name and click **Create App**
+  1. Go to the app's configuration by clicking **View Your App**.
+  1. Scroll to the **Application Scopes** section of the same screen
+     and ensure that at least the following scopes are enabled:
+     * Read and write all files and folders stored in Box
```

Comment:
Markdown isn't rendering this as a nested list. Maybe give this a preview in your editor?

Reply:
hmmm Macdown is rendering it as a nested list. Let me try spacing the lists and see if it picks up properly. If that doesn't work I'll rework the wording to avoid the nesting.

Reply:
> ```
> graph LR
>   a[Okta login] --> b{Has Matching Box Account?}
>   b{Has Matching Box Account?} -- Yes --- c[Retrieve Box User]
>   b{Has Matching Box Account?} -- No --- d[Create new Box App User]
> ```

Did you wrap this in a code block?

~~~
```mermaid
graph LR
  a[Okta login] --> b{Has Matching Box Account?}
  b{Has Matching Box Account?} -- Yes --- c[Retrieve Box User]
  b{Has Matching Box Account?} -- No --- d[Create new Box App User]
```
~~~

Reply:
I managed to fix this. It was our formatter that was acting up.